### PR TITLE
Add onepasswordItemFields template function

### LINKED ISSUE #1098: 1password op get item --fields support
## Describe the solution you'd like

Current `op` command has a `get item` with `--fields` flag which makes it a lot simpler to extract values from details or **section fields**.

The `output` function can be used with this:

```
{{- output "op" "get" "item" "some-id" "--fields" "some-field" }}
```

But it might be nicer to have a built-in _onepassworditemFields_ function (or something) to make this a bit nicer to integrate?

### PR DESCRIPTION
**Pull request by twpayne**
Fixes #1098.

### CODE REVIEW

**Comment by bric3** [comment]
Neat !

File: docs/REFERENCE.md [hunk 1]
```diff
@@ -2116,6 +2117,57 @@ accounts).
...
+        "k": "string",
+        "n": "D4328E0846D2461E8E455D7A07B93397",
+        "t": "exampleLabel",
+        "v": "exampleValue"
+    }
+}
+```
+
+---
+
```
Comment (bric3) [inline docs/REFERENCE.md:2170]: I think the readme should provide the equivalent commands

```shell
op get item abcdefghijklmnopqrstuvwxyz --fields "exampleLabel"
exampleValue
```

```go-template
{{ (onepasswordItemFields "abcdefghijklmnopqrstuvwxyz").exampleLabel.v }}
```

**Comment by bric3** [comment]
Hi,

After looking at the code I wonder if this approach is future proof in the sens that 1password could change their structure payload. Why not using the command flag `--fields` (of course it means chezmoi will require a minimum version of the `op` tool).

File: docs/REFERENCE.md (hunk 1 above)
Comment (twpayne) [reply docs/REFERENCE.md:2170]: Added in a79ba2f0699ddbc1245715da12975daec18db513.

**Comment by twpayne** [comment]
If 1password change the structure of the return value they will break a lot of things for other people, so I don't think they would do it. The reason for using the JSON output rather than the command flag is that the JSON output has a well-defined format and multiple requests for different aspects of the same item only result in one underlying call to `op`.

**Comment by bric3** [comment]
Indeed this is a remote possibility. Note that I don't actually have preference in term of usage.

**Pull request merged**